dirty.html

```diff
@@ -198,8 +229,14 @@
       </div>
       <div></div>
     </div>
-    <div id="footer">
+    <!-- Error, the div id=footer can just be a footer tag, it also had no closing tag -->
+    <footer>
       <div>
-        <div id="span">Web Development Immersive Course Webpage</div> General Assembly London <div id="span">2013</div id="span">
+      <!-- Error, multiple id span is scene, perhaps a class should be assigned to these instead of id as id is to be a unique reference. there is also a div id name to end a div, that should be removed-->
+        <div class="span">Web Development Immersive Course Webpage</div> General Assembly London <div class="span">2013</div>
```

Comment:
`div`s within `div`s should be in separate lines... each `<div>` opens a block element, so we need to tell that to other users who read the code by indenting it accordingly... `<span>` on the other hand, so we can put it in the same line...